modules/rostests/apitests/ntdll/NtWriteFile.c

```diff
@@ -56,6 +59,87 @@ SizeOfSector(VOID)
     return SectorSize;
 }
 
+void test_FileAppendData(void)
+{
+    NTSTATUS Status;
+    HANDLE FileHandle;
+    IO_STATUS_BLOCK Iosb;
+    FILE_POSITION_INFORMATION FilePosInfo;
+    OBJECT_ATTRIBUTES ObjectAttributes;
+    CHAR chBuffer[256];
+    ULONG uDataLen;
+    UNICODE_STRING FileName;
+    RtlInitUnicodeString(&FileName, L"\\??\\C:\\Documents and Settings\\test.txt");
```

Comment:
You should try investigate (some other apitests may do that) where to create that test file either in the root directory or somewhere in a temporary directory.